scripts/generate_workers_cfg.sh

```diff
@@ -94,7 +94,7 @@ generate_workers () {
               shared_volumes=$(echo ${SHARED_WORK_DIRECTORIES} | tr ";" "\n")
               for shared_volume in $shared_volumes
               do
-                  add_item ${shared_volume}
+                  add_item ${shared_volume}:/data
```

Comment:
revert that, you've broken the multi-folder workdir.
And it's out of the scope of this PR !